packages/build-tools/src/ios/fastlane.ts

```diff
@@ -102,13 +103,18 @@ async function ensureGymfileExists<TJob extends Ios.Job>(
 
   ctx.logger.info('Creating Gymfile');
   if (ctx.job.simulator) {
+    const simulatorDestination = isTVOS(ctx, buildConfiguration)
+      ? 'generic/platform=tvOS Simulator'
+      : 'generic/platform=iOS Simulator';
```

Comment:
If I were writing this, I'd just pass `tvOS`/`iOS` top the Gymfile template.

Reply:
I agree, given that it's not obvious what scheme and buildConfiguration value should be used, it's probably best to resolve ios vs tvos in one place and pass that value where it's needed e.g. in the main builder code https://github.com/expo/eas-build/blob/main/packages/build-tools/src/builders/ios.ts

Reply:
This is now expressed in a nicer way